pkg/standalone-utils/contracts/LidoRelayer.sol

```diff
@@ -71,21 +71,26 @@ contract LidoRelayer is RelayerAssetHelpers, ReentrancyGuard {
             uint256 wstETHAmount = singleSwap.kind == IVault.SwapKind.GIVEN_IN ? singleSwap.amount : limit;
             _pullStETHAndWrap(msg.sender, wstETHAmount);
             _approveToken(IERC20(address(_wstETH)), address(getVault()), wstETHAmount);
+
+            swapAmount = getVault().swap{ value: msg.value }(singleSwap, funds, limit, deadline);
+
+            // GIVEN_OUT trades can leave an unknown amount of wstETH
+            // We then must refund the full relayer balance
+            _unwrapAndPushStETH(msg.sender, IERC20(address(_wstETH)).balanceOf(address(this)));
```

Comment:
We could put this behind an `if kind == given out` and skip the `balanceOf` call, no?